packages/account/src/Components/poinc/poinc-file-uploader-container/poinc-file-uploader-container.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { Icon } from '@deriv/components';
+import { isDesktop, WS } from '@deriv/shared';
+import { Localize } from '@deriv/translations';
+import FileUploader from '../../file-uploader';
+
+const PoincFileUploaderContainer = ({ is_description_enabled = true, getSocket, onFileDrop, onRef }) => {
+    const ref = React.useRef();
+
+    const getSocketFunc = getSocket ?? WS.getSocket;
+
+    React.useEffect(() => {
+        if (ref) {
+            if (typeof onRef === 'function') onRef(ref);
+        }
```

Comment:
You can simplify this.
```suggestion
        if (ref && typeof onRef === 'function' )  onRef(ref);
```

Reply:
good note, thanks, done :+1: